resources/views/technique/index.blade.php

```diff
@@ -0,0 +1,17 @@
+@extends('layouts.app')
+@section('title', $viewData['title'])
+@section('content')
+@foreach ($viewData['technique'] as $technique) 
+<div class="col-md-4 col-lg-3 mb-2" style="margin: 15px;"> 
+    <div class="card" style="width: 18rem;"> 
+        <img src="{{ $technique['image'] }}" class="card-img-top" alt="{{ $technique['model'] }}"> 
```

Comment:
Lo cambiaría a esto (para utilizar solo getters)
```suggestion
        <img src="{{ $technique-getImage() }}" class="card-img-top" alt="{{ $technique->getModel() }}"> 
```